### PR TITLE
Updating chatmessage model

### PR DESCRIPTION
**Pull request by patrisiakaplun**
Updating chatmessage model after adding client model and profession: 
Updating models.py, 0002_test_data.py and tests.py files Adding 0002_chatmessage_sender_type_alter_chatmessage_client_id_and_more.py file

resolves issue #46

### CODE REVIEW

File: chatmessage/tests.py [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-PROFESSIONAL_ID = "professional1"
-CLIENT_ID = "client1"
+
 MESSAGE = "message1"
 
 
 @pytest.fixture
 def chatmessage():
-    return Chatmessage(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, message=MESSAGE)
+    return Chatmessage(professional_id=Professional.filter_by_professional_id(1)[0],
```
[the remark this answers is not in the document]
Comment (patrisiakaplun) [reply chatmessage/tests.py]: done

**Review by guybeckenstein — APPROVED** [review]
LGTM

File: chatmessage/tests.py [hunk 2]
```diff
@@ -1,29 +1,30 @@
...
+
 MESSAGE = "message1"
 
 
 @pytest.fixture
-def chatmessage():
-    return Chatmessage(professional_id=PROFESSIONAL_ID, client_id=CLIENT_ID, message=MESSAGE)
+def chatmessage(professional, client):
+    return Chatmessage(professional_id=professional,
+                       client_id=client, message=MESSAGE, sender_type=SenderType.Client)
```
Comment (idosinger98) [inline chatmessage/tests.py:12]: I think you should add it to conftest.py

File: chatmessage/tests.py [hunk 3]
```diff
@@ -1,29 +1,30 @@
...
 
 
 @pytest.fixture
 def persisted_chatmessage(chatmessage):
     chatmessage.save()
     return [(chatmessage.message_id), (chatmessage.professional_id),
-            (chatmessage.client_id), (chatmessage.date), (chatmessage.message)]
+            (chatmessage.client_id), (chatmessage.date), (chatmessage.message), (chatmessage.sender_type)]
```
Comment (idosinger98) [inline chatmessage/tests.py:19]: I think you should add it to conftest.py

File: chatmessage/tests.py (hunk 2 above)
Comment (patrisiakaplun) [reply chatmessage/tests.py]: Done

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**